databricks/koalas/internal.py

```diff
@@ -793,12 +793,26 @@ def to_pandas_frame(self) -> pd.DataFrame:
             ]
         return pdf
 
+    @lazy_property
+    def applied(self):
```

Comment:
Should we maybe call it like:
- `resolved_copy`
- `applied_copy`
- `new_sdf_copy`
- ...

?

The name `applied` doesn't look clear that it's going to have a new Spark DataFrame internally that changes anchor.